src/screens/mpesaToCusd/AddFunds.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { useNavigation, useRoute } from "@react-navigation/core";
+import { TextInputMask } from "react-native-masked-text";
+import { LinearGradient } from "expo-linear-gradient";
+import ScreenComponent from "../../containers/ScreenComponent";
+import NavHeader from "../../containers/NavHeader";
+import KeyPad from "../../components/Buttons/KeyPad";
+import COLORS from "../../styles/colors/colors";
+
+const AddFunds: React.FunctionComponent = () => {
+  const route = useRoute();
+  const navigation = useNavigation();
+
+  const [value, setValue] = useState("");
+  // const operation = route.params.operation;
+
+  function handleChange(newValue) {
+    setValue(newValue);
+  }
+
+  return (
+    <ScreenComponent>
+      <NavHeader
+        showTitle={true}
+        //   newTitle={operation === "TopUp" ? "Add Funds" : "Withdraw Funds"}
+      />
+      <View style={styles.container}>
+        <TextInputMask
+          type={"money"}
+          options={{
+            unit: "Ksh ",
+          }}
+          value={value}
+          style={styles.title}
+          placeholder="Ksh 0,00"
+          placeholderTextColor={COLORS.primary}
+        />
+        <KeyPad value={value} onChange={handleChange} />
+        <TouchableOpacity
+        // onPress={() =>
+        //   navigation.navigate("Confirm Funds", {
+        //     value: value,
+        //     operation: operation,
+        //   })
+        // }
+        >
+          <LinearGradient
+            colors={["rgba(183, 0, 76, 0.3)", "rgba(19, 63, 219, 1)"]}
+            start={[1, 0]}
+            end={[0, 1]}
+            style={styles.button}
+          >
+            <Text style={styles.buttonText}>Review</Text>
+          </LinearGradient>
+        </TouchableOpacity>
+      </View>
+    </ScreenComponent>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginHorizontal: 30,
+    justifyContent: "flex-end",
+  },
+
+  title: {
```

Comment:
@Alphonce-Mutebi there is also a bunch of styling information from here on onward until the end of the file.

font size
color
font family
why is that? should that better be imported?